crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -19,6 +19,7 @@
 require "json"
 
 class BarclampController < ApplicationController
+  skip_before_filter :enforce_installer
```

Comment:
Oh, sorry, I misread and didn't see the `skip`.

Hrm. It's slightly more complicated, though: if chef is not started yet, then this controller will likely fail. We probably need to allow this controller only when we're at the step where it will really work.